draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1894,6 +1894,11 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
     as the comma can only appear inside quoted strings;
     bad example: "Location", as a comma can occur inside a URI).</t>
   </li>
+  <li>
+    <t>When it does use list syntax, this needs to happen at the top-level; that is,
```

Comment:
"needs to" is a bit weird here -- maybe "is encouraged to"? Also, describe why it's bad?